cmd/ui/src/views/Explore/ExploreSearch/CypherSearch.tsx

```diff
@@ -63,6 +61,21 @@ const useStyles = makeStyles((theme) => ({
         maxHeight: '500px',
         overflow: 'auto',
     },
+    cypherEditorDark: {
+        display: 'flex',
+        flexGrow: 1,
+        flexDirection: 'column',
+        border: '1px solid',
+        borderColor: 'rgba(0,0,0,.23)',
+        borderRadius: theme.shape.borderRadius,
+        backgroundColor: '#002b36',
+        paddingTop: '5px',
+        minHeight: '120px',
+        // enables drag n drop resizing of editor
+        resize: 'vertical',
+        maxHeight: '500px',
+        overflow: 'auto',
```

Comment:
I believe this `overflow` was just removed in BED-4365 - we remove that here (and for the light mode version too.)

Reply:
Good catch, thank you for pointing this out.